breakfast/readme.md

```diff
@@ -1,21 +1,44 @@
-# Replace with recipe name
+# Pancakes
 
-## Servings: 
+## Servings: 3 servings
 
-## Time to prepare: 
+## Time to prepare: 30 minutes
 
 ## Ingredients:
-
+1 Egg
+1 cup of milk
+Pancake mix
+Tablespoon of olive oil
+Cinnamon
+BBQ Sauce
```

Comment:
Think we should remove BBQ sauce from the list of ingredients

Reply:
Agreeeed

Reply:
Let's start a review